Makefile

```diff
@@ -12,183 +9,58 @@ DATE_FMT = +%Y%m%d%H%M.%S
 # Fallback mechanism to support other systems:
 # 1. 'date -d': Busybox and GNU coreutils.
 # 2. 'date -r': BSD date. It does not support '-d'.
-BUILD_DATE = $(shell date -u -d "@${SOURCE_DATE_EPOCH}" "${DATE_FMT}" 2>/dev/null || date -u -r "${SOURCE_DATE_EPOCH}" "${DATE_FMT}")
+export BUILD_DATE = $(shell date -u -d "@${SOURCE_DATE_EPOCH}" "${DATE_FMT}" 2>/dev/null || date -u -r "${SOURCE_DATE_EPOCH}" "${DATE_FMT}")
+
+
+date:
+	@echo $(BUILD_DATE)
 
-GO_BUILDFLAGS ?= -trimpath -ldflags="-s -w"
+clean:
+	@rm -rf dist/
+	@docker image ls "$(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME)*" -aq | xargs docker rmi --force
 
-ifndef GOARCH
-	GOARCH=amd64
-endif
+dist:
+	@goreleaser release --snapshot --rm-dist
```

Comment:
```suggestion
	@go run github.com/goreleaser/goreleaser@v1.14.1 release --snapshot --rm-dist
```

Reply:
Ah nice, I was not aware of this. Even though, I saw it in some other commands.

https://github.com/elastic/apm-aws-lambda/pull/362/commits/8bd2e4302b57a015a963ec39b16642bc946df466